js/navbar/menuButton.js

```diff
@@ -1,3 +1,5 @@
+var keybindings = require('keybindings.js')
```

Comment:
This was a pre-existing issue where `keybindings` was being declared from taskOverlay, which was wrong - with taskOverlay as a module, that doesn't happen anymore.